src/Test/Runner/Log.elm

```diff
@@ -0,0 +1,76 @@
+module Test.Runner.Log exposing (run, runWithOptions)
+
+{-| # Log Runner
+
+Runs a test and outputs its results using `Debug.log`, then calls `Debug.crash`
+if there are any failures.
+
+This is not the prettiest runner, but it is simple and cross-platform. For
+example, you can use it as a crude Node runner like so:
+
+    $ elm-make LogRunnerExample.elm --output=elm.js
+    $ node elm.js
+
+This will log the test results to the console, then exit with exit code 0
+if the tests all passed, and 1 if any failed.
+
+@docs run, runWithOptions
+-}
+
+import Random
+import Test exposing (Test)
+import Test.Runner.String exposing (Summary)
+import String
+
+
+{-| Run the test using the default `Test.Runner.String` options.
+-}
+run : Test -> a -> a
```

Comment:
I ended up just moving the Log and String runners into `examples`. This way people can still try quickly things out with `LogRunnerExample` if they like.

I didn't bother updating the API to use a `Task`, since it's no longer publicly exposed.